app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt

```diff
@@ -91,6 +94,9 @@ class CompletedStoryListActivityTest {
   private val internalProfileId = 0
   private val profileId = ProfileId.newBuilder().setInternalId(internalProfileId).build()
 
+  @get:Rule
+  val activityTestRule = ActivityTestRule(CompletedStoryListActivity::class.java)
```

Comment:
Suggest adding the other params too: https://github.com/oppia/oppia-android/blob/75c25b8094a9a84197397cd40887a491f8118d0f/app/src/sharedTest/java/org/oppia/android/app/faq/FAQSingleActivityTest.kt#L90

Reply:
done